k8s-daemonset/k8s-ingress/nginx.yml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: nginx
+spec:
+  replicas: 1
+  selector:
+    app: nginx
+  template:
+    metadata:
+      name: nginx
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - name: nginx
+        image: buoyantio/helloworld:nginx4
```

Comment:
Maybe we should call this image `buoyant/nginx-more`?  Would also be good to check the Dockerfile for that image into the `docker` top-level directory in this repo.